proposals/0002-code-gallery.md

```diff
@@ -0,0 +1,133 @@
+    - Switches from `EntityComponent` -> `SpawnableEntityComponent`, `MobileEntityComponent` without mentioning it
+- Address [Tyler friend project feedback](https://discord.com/channels/931210784011321394/1166512616718471279/1196276328543031376)
+    - It was not clear from the docs that we had to install a unity package, and it was not clear on the website where to get that
+    - Connecting to the server with the Unity package was not obvious how to do
+    - It was not clear that we even had to use the `NetworkManager`
+    - Why do we call row insert for populating the client cache?
+
+## Create docs section: “How do I…?” Code demo gallery
+
+- Add this new section at the end of the Unity tutorial, for demoing individual game features and accompanying code
+- Each demo should have its own GitHub PRs or branches, with a shared “base project” where the Unity tutorial leaves off (or a previous demo).
+- Each page should make clear
+    - Motivating feature or use case (e.g. inventory system, resource mining)
+    - MVP code to add the feature, with appropriately generic names not tied to any specific broader game etc.
+    - Explain the “key pieces” of the code
+    - Include links to other demos or the tutorial for further reading
+- Move advanced functionality from the current Unity tutorial, into self-contained “How do I…?” pages listed in the navbar:
+    - Chat functionality
+    - Inventory system, management
+    - Player-environment interaction
+    - Shop functionality
+    - Resource spawning, mining
+- Add a CTA button, something like “Let us know if we’re missing your use case!”
+
+# Execution stages
+
+I propose to approach things in this order:
+
+1. Update project to latest SpacetimeDB (since this might affect the set of errata)
+2. Address tutorial docs quick fixes / errata (pretty well-contained scope, high visibility/impact)
+3. Simplify code using: https://github.com/clockworklabs/zeke-demo-project
+4. Greybox out the meshes (I believe this depends on some animator code being simplified)
+5. Copy (parts of) tutorial code into their own self-contained branches
+6. Create Markdown pages for each one, create the new docs section
+
+# Alternatives considered
+
+- Maintaining one sequential tutorial that progressively demonstrates each topic / use case
+    - I claim this is harder to do well, is more difficult to maintain and test
+- Redo the tutorial “from scratch” with a game more focused on highlighting SpacetimeDB features:
+    - This would be helpful for e.g. attractive demoability on a stream
+    - Switching to a meaningfully different design would require a meaningful amount of work “figuring out details” again (even to a minimal design)
+    - Currently the most important thing is that users are able to understand the basic concepts, and I believe the current tutorial largely achieves that
+
+# Open questions
+
+- Name for the new docs section?
+    - “How do I…?”
+    - “Code Demo Gallery”
```

Comment:
I would consider the universally-popular "Frequently Asked Questions."

If these will specifically be for features added to the Unity tutorial game, I would consider the more-specific "Add-on Features."